lib/landlord/engine.rb

```diff
@@ -0,0 +1,7 @@
+module Landlord
+  class Engine < Rails::Engine
+    rake_tasks do
+      load File.join(__dir__, "tasks/landlord.rake")
+    end
+  end
+end
```

Comment:
with this class we can share all we have within app/ with the actual app.

And rake_tasks does what it says: loads the rake tasks defined in tasks/landlord.rake